src/commands/help.js

```diff
@@ -6,23 +6,35 @@ import {stdtermwidth} from 'cli-engine-command/lib/output/screen'
 import Plugins from '../plugins'
 import type Plugin from '../plugins/plugin'
 
+function trimMax (n: number) : number {
+  // 3/5 of stdtermwidth is an assumption here
```

Comment:
you know I think we could calculate the exact width, which I think would be ideal. I'll take a look tomorrow to see how we might be able to do that